spec/controllers/organizations_controller_spec.rb

```diff
@@ -0,0 +1,104 @@
+require 'spec_helper'
+
+describe OrganizationsController do
```

Comment:
I forgot to change the tests as well after making suggested changes. I'll fix this.